openedx/core/lib/request_utils.py

```diff
@@ -80,3 +88,32 @@ def course_id_from_url(url):
         return CourseKey.from_string(course_id)
     except InvalidKeyError:
         return None
+
+
+class CookieMetricsMiddleware(MiddlewareMixin):
```

Comment:
Want to use a feature toggle - just in case we need to turn this off for some reason?

Reply:
Added. Please check? I verified it worked locally, but I can't remember the last time I used waffle.